ports/espressif/boards/nodemcu_esp32c2/board.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ *
+ * The MIT License (MIT)
```

Comment:
Headers have been updated to SPDX. Is this based on an old branch?

Reply:
`espressif_esp8684_devkitc_02_n4` which I used as a base, wasn't updated apparently.
Updated.

Reply:
Ok, thanks! I'll up date it too.